internal/handlers/common/update.go

```diff
@@ -36,9 +36,9 @@ func UpdateDocument(doc, update *types.Document) (bool, error) {
 	if update.Len() == 0 {
 		// replace to empty doc
 		for _, key := range doc.Keys() {
-			if key != "_id" {
-				changed = true
+			changed = true
```

Comment:
Compat test tells if there is any key including `_id`, it should return true for changed. Added `Unset` to shareddata and this change passes compat test for `Unset`